### ISSUE TITLE
Why does the transfer-object robot verify that versionMetadata exists?

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/preservation_robots#348]
https://github.com/sul-dlss/preservation_robots/blob/main/lib/robots/sdr_repo/preservation_ingest/transfer_object.rb#L33

**Comment by justinlittman** [comment]
Is there an action on this ticket?

**Comment by jcoyne** [comment]
I think we want to figure out why versionMetadata.xml is the target and if we can switch it to be something else, because we are probably not going to generate versionMetadata.xml in the future.

**Comment by jmartin-sul** [comment]
here's what i found when looking into this yesterday:
- i don't know that i can see why it's needed.  the only references to it in pres robots are in `lib/deposit_bag_validator.rb` and `lib/robots/sdr_repo/preservation_ingest/transfer_object.rb`
  - seems like we just really want to be sure we have it?  it's the only fedora datastream among the `REQUIRED_BAG_FILES` - https://github.com/sul-dlss/preservation_robots/blob/main/lib/deposit_bag_validator.rb#L45-L54
- i don't see any reference to it in the moab-versioning gem either.  seems like moab-versioning doesn't care, as long as there's a valid bag to work with.
- hard to say what the intent was, whether it was just a proxy for "did the datastreams get exported?" (under the assumption that one's always there? sometimes seen missing? 🤷)  or something else entirely
- maybe we should test accessioning and versioning an object without versionMetadata.xml, on a branch w/ the check disabled, to see if it works
- as @jcoyne points out, there's a check done on DSA side already, and the check from pres robots is _before_ the tarpipe to the deposit dir, it just `ssh`es to dor-services to do the check that DSA already just did (so this is not analogous to pres cat verifying a new moab version after it's been created by copying files from the deposit bag to the new moab version, which involves bits being written anew from the deposit dir to the storage root)
- people to ask if we really wanted to understand the intent: ben, lynn, richard anderson

**Comment by ndushay** [comment]
@jcoyne - are we still generating versionMetadata.xml?

What should we use instead?   can we use the cocina json file?  Or ???

**Comment by jcoyne** [comment]
Yes, we are still generating versionMetadata.xml, but mostly because we have this check.  I think switching the check to look for `bagit.txt` would be better (cocina.json would also work). But I'm not sure we really need this check at all because we already have a validator that ensures the bag has all the required files.
